### PR TITLE
feat: ask me anything

### PR DESCRIPTION
**Pull request by djaiss**
- [x] docs
- [x] update Help icon link
- [x] state when no AMA on the Company HR tab
- [x] delete an AMA session

### CODE REVIEW

File: resources/js/Pages/Company/HR/AskMeAnything/Answered.vue [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+          </h2>
+
+          <!-- session theme -->
+          <p v-if="data.theme" class="mb3 tc mv0">{{ data.theme }}</p>
+        </div>
+
+        <!-- tabs -->
+        <div class="cf mw7 center br3 mb5 tc">
+          <div class="cf dib btn-group">
+            <inertia-link :href="data.url.unanswered_tab" class="f6 fl ph3 pv2 dib pointer no-underline" :class="{'selected':(tab == 'unanswered')}">
```
Comment (asbiin) [inline resources/js/Pages/Company/HR/AskMeAnything/Answered.vue:45]: You can remove quotes and parenthensis, is easier to read
```suggestion
            <inertia-link :href="data.url.unanswered_tab" class="f6 fl ph3 pv2 dib pointer no-underline" :class="{ selected: tab === 'unanswered' }">
```

File: resources/js/Pages/Company/HR/AskMeAnything/Answered.vue [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+              {{ $t('company.hr_ama_show_questions_tab_unanswered') }}
+            </inertia-link>
+            <inertia-link :href="data.url.answered_tab" class="f6 fl ph3 pv2 dib pointer" :class="{'selected':(tab == 'answered')}">
```
Comment (asbiin) [inline resources/js/Pages/Company/HR/AskMeAnything/Answered.vue:48]: same
```suggestion
            <inertia-link :href="data.url.answered_tab" class="f6 fl ph3 pv2 dib pointer" :class="{ selected : tab === 'answered' }">
```

File: resources/js/Pages/Company/HR/AskMeAnything/Index.vue [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+          <div class="tc">
+            <inertia-link :href="data.active_session.url" class="btn db mb2">{{ $t('app.view') }}</inertia-link>
+            <p class="mv0 gray f7">{{ $tc('company.hr_ama_index_next_questions', data.active_session.questions_count, {count: data.active_session.questions_count }) }}</p>
+          </div>
+        </div>
+        <div v-else class="br3 mb5 bg-white box flex justify-between items-center pa3">
+          <div class="relative flex items-center">
+            <h2 class="fw4 f5 mt0 mb0 tc">
+              There are no planned scheduled sessions.
```
Comment (asbiin) [inline resources/js/Pages/Company/HR/AskMeAnything/Index.vue:64]: message to localize

File: resources/js/Pages/Company/HR/AskMeAnything/Show.vue [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+        <div v-if="data.permissions.can_edit" class="actions pa3 box flex justify-center mb4">
+          <inertia-link :href="data.url.edit" class="btn dib mr3">{{ $t('app.edit') }}</inertia-link>
+          <a v-if="activeStatus" class="btn" @click="toggle()">{{ $t('company.hr_ama_deactivate') }}</a>
+          <a v-else class="btn" @click="toggle()">{{ $t('company.hr_ama_activate') }}</a>
+        </div>
+
+        <!-- tabs -->
+        <div class="cf mw7 center br3 mb5 tc">
+          <div class="cf dib btn-group">
+            <inertia-link :href="data.url.unanswered_tab" class="f6 fl ph3 pv2 dib pointer no-underline" :class="{'selected':(tab == 'unanswered')}">
```
Comment (asbiin) [inline resources/js/Pages/Company/HR/AskMeAnything/Show.vue:87]: same
```suggestion
            <inertia-link :href="data.url.unanswered_tab" class="f6 fl ph3 pv2 dib pointer no-underline" :class="{ selected: tab === 'unanswered' }">
```

File: resources/js/Pages/Company/HR/AskMeAnything/Show.vue [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+              {{ $t('company.hr_ama_show_questions_tab_unanswered') }}
+            </inertia-link>
+            <inertia-link :href="data.url.answered_tab" class="f6 fl ph3 pv2 dib pointer" :class="{'selected':(tab == 'answered')}">
```
Comment (asbiin) [inline resources/js/Pages/Company/HR/AskMeAnything/Show.vue:90]: same
```suggestion
            <inertia-link :href="data.url.answered_tab" class="f6 fl ph3 pv2 dib pointer" :class="{ selected: tab === 'answered' }">
```

**Review by asbiin — CHANGES_REQUESTED** [review]
It's great! See my changes proposal

File: resources/js/Pages/Company/HR/AskMeAnything/Answered.vue (hunk 1 above)
Comment (djaiss) [reply resources/js/Pages/Company/HR/AskMeAnything/Answered.vue:45]: I didn't know that!!!

File: resources/js/Pages/Company/HR/AskMeAnything/Index.vue (hunk 3 above)
Comment (djaiss) [reply resources/js/Pages/Company/HR/AskMeAnything/Index.vue:64]: Good catch

**Pull request merged**